acsoo/addons.py

```diff
@@ -85,3 +103,57 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+def do_makepot(config, database, odoo_bin, addons_dirs, odoo_config):
+    cmd = [
+        odoo_bin,
+        'shell',
+        '-d', database,
+        '--log-level=info',
+        '--no-xmlrpc',
+    ]
+    if odoo_config:
+        cmd.extend([
+            '-c', odoo_config
+        ])
+    script_dir = os.path.dirname(__file__)
+    file_path = os.path.join(script_dir, 'make_pot_script')
+    proc = subprocess.Popen(
+        cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT)
+    with open(file_path) as file:
+        cmd = file.read()
+    installable_addons = get_installable_addons(addons_dirs).items()
+    for addon, addon_dir in installable_addons:
```

Comment:
`for addon_name, (addon_dir, manifest) in installable_addons`

See also https://github.com/acsone/acsoo/pull/20/files#r164405720